website/assets/scss/common/_cards.scss

```
@@ -330,7 +330,7 @@
 .section-title.section-title--left {
   text-align: left;
   font-weight: 700;
-  font-family: 'Inter var', 'Inter', system-ui, sans-serif;
+  font-family: 'Inter Variable', 'Inter var', 'Inter', system-ui, sans-serif;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)